docs/node-operation/itn-manual.md

```diff
@@ -0,0 +1,210 @@
+---
+id: ITN Manual
+sidebar_position: 4
+slug: /itn-manual
+---
+
+# Incentivized Testnet manual
```

Comment:
I wonder if it makes sense to call this an overview, and then have a separate manual section?

The manual section would just be a pointer to:

* manual ITN phase 1 <link>
* manual ITN phase 2 <link>
* manual ITN phase 2 <link>
* and so on...

And each of the manual pages would just have a title and then a message along the lines of "more info to be provided as we approach the ITN phase"?

Reply:
@ross-spencer 
To be clear, are you suggesting a structure like this:
```
├─Introduction
├─Orcfax solution
├─Fact token
├─FAQ
├─ITN
    │   ├─ Overview // the current page being reviewed
    │   ├─ Phase 1 manual // link out to from overview "phase #" sections?
    │   ├─ Phase 2 manual
    │   ├─ Phase 3 manual
    │   ├─ Phase 4 manual
├─Node operation
└─Developer manual
```

Reply:
That layout looks good!

Reply:
It looks like this page has been replaced with the overview now (and repeats much of the same information?), do we want to delete it?

Reply:
a mindless hang-over
deleted!